Bundle/BusinessEntityBundle/Annotation/AnnotationDriver.php

```diff
@@ -88,7 +88,6 @@ public function getAllClassNames()
             );
             foreach ($iterator as $file) {
                 $sourceFile = realpath($file[0]);
-                require_once $sourceFile;
```

Comment:
Doctrine does this "require_once" when the doctrine-bundle boots. If your cache:warmup works well even if you declare the DoctrineBundle under the BusinessEntityBundle in your AppKernel, then we can indeed remove this line.